platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/utils/FileUtils.kt

```diff
@@ -0,0 +1,75 @@
+package org.maplibre.android.utils
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import org.maplibre.android.log.Logger
+import java.io.File
+
+object FileUtils {
+	private const val TAG = "Mbgl-FileUtils"
+
+	/**
+	 * Deletes a file asynchronously in a separate thread.
+	 *
+	 * @param path the path of the file that should be deleted
+	 */
+	fun deleteFile(path: String) {
+		// Delete the file in a separate thread to avoid affecting the UI
+		Thread {
```

Comment:
Inconsistency. You should use a coroutine with IO dispatcher.

Reply:
Missed this because I was only dealing with `AsyncTask` calls.  I'll fix it to be consistent.